### ISSUE TITLE
Bash completion displays hidden and deprecated flags

### ISSUE THREAD

**Issue by nak3** [openshift/origin#19256]
Upstream fixed it 2 years ago - https://github.com/spf13/cobra/pull/313, but origin still has the issue (because origin is using older version) https://github.com/openshift/origin/blob/35ba454e125c5b46b8848febfd68f7f3c858ad94/glide.lock#L1064-L1065

### Steps to reproduce (Following flags are the examples)

```
$ oc new-build --build-
--build-arg=       --build-env=       --build-env-file=  --build-secret=
                        ^                  ^
			+--marked hidden	   +--marked hidden

$  $ oc new-app --image
--image=         --image-stream=
   ^
   +--- deprecated
```

**Comment by php-coder** [comment]
@openshift/sig-developer-experience

**Comment by nak3** [comment]
Yes, the cobra was updated by https://github.com/openshift/origin/pull/19137 last month. Thank you.

> I'd suggest you verify if the correct completions are properly installed on your system.

I installed and used correct completions. But the https://github.com/openshift/origin/pull/19137 was merged between my report and your verify.. :cry:
